### ISSUE TITLE
vuedevtools vuex tab does not work via `quasar build`

### ISSUE THREAD

**Issue by geoidesic** [quasarframework/quasar#7911]
**Describe the bug**

If you browse a prod SPA app built with `quasar build`, the vuex tab of vuexdevtools will not work. You get the message `No vuex store detected`, whereas in a vanilla VueJS build it works.

**Codepen/jsFiddle/Codesandbox (required)**
It is not possible to demonstrate this bug on these platforms. The bug ONLY OCCURS if you `quasar build` and then serve that via a web server and browse to it that way.
None the less I've shown the code required here: https://codesandbox.io/s/vuedevtools-vuex-tab-setup-31v4l

**To Reproduce**
1. In both vanilla Vue and Quasar it's possible to set up vuedevtools to work on prod. (i.e. `quasar build`). In vanilla vue, you just need the `Vue.config.devtools`.  In Quasar this requires a boot file like this:
```
import { boot } from "quasar/wrappers";
export default boot(({ Vue }) => {
  Vue.config.devtools = true;
});
```
1. Then this must be enabled in `quasar.conf.js`:
```
    boot: ['devtools'],
```
1. Enable the default vuex store by uncommenting lines 4 & 20 in `src/store/index.js`
1. Add some state to the store in `src/store/example/state.js`
1. Then you need to build it: `quasar build` 
1. Then you need to serve that via a web server (exercise for reader)
1. Then browse to it
1. Open devtools, click vue tab to get vuedevtools
1. Click on the `vuex` tab in vuedevtools
1. Click on the `Base State` mutation to load the state into the `vuex` viewer. 

**Expected behavior**
It should show the vuex state

**Screenshots**
Two screenshots. One shows it working via localhost (i.e. via `quasar dev`) 
The other shows it not working via `quasar build`
![Screenshot 2020-10-16 at 14 56 15](https://user-images.githubusercontent.com/4637995/96267520-cbf60400-0fbf-11eb-8548-5298627d2b4e.png)
![Screenshot 2020-10-16 at 14 55 59](https://user-images.githubusercontent.com/4637995/96267522-cd273100-0fbf-11eb-8331-62eb34af59ca.png)

**Platform (please complete the following information):**
OS: Mac OS Catalina 10.15.7 
Node: v12.6.0
NPM: 6.14.8
Yarn: 1.22.4
Browsers: Chrome Version 86.0.4240.80 (Official Build) (x86_64)
iOS: N/A
Android: N/A
Electron: N/A

**Additional context**

**Comment by geoidesic** [comment]
According to this issue: https://github.com/vuejs/vue-devtools/issues/391 Vue.config.devtools = true  needs to happen before Vue.use(Vuex). Looking at the built js file I think Quasar does it the other way around.

**Comment by geoidesic** [comment]
Confirmed. This is the problem. If I edit the built JS file directly and put it in that way around, then it works. So this is definitely a bug.

**Comment by geoidesic** [comment]
My suggestion is that this shouldn't require a boot file. This should be a `quasar.conf.js` setting. There should be a section in `quasar.conf` `build` section for setting Vue `config` settings. These can then just be looped through and set before any `Vue.use()` calls.

**Comment by geoidesic** [comment]
I found a way. This setting can be made in the `/store/index.js` file.

```
import Vue from "vue";
import Vuex from "vuex";

import example from "./module-example";

Vue.config.devtools = true;   <--- here

Vue.use(Vuex);

/*
 * If not building with SSR mode, you can
 * directly export the Store instantiation;
 *
 * The function below can be async too; either use
 * async/await or return a Promise which resolves
 * with the Store instance.
 */

export default function(/* { ssrContext } */) {
  const Store = new Vuex.Store({
    modules: {
      example
    },

    // enable strict mode (adds overhead!)
    // for dev mode only
    // strict: process.env.DEV
    strict: true
  });

  return Store;
}
```

**Comment by hawkeye64** [comment]
@geoidesic When this is released, all you need to do for getting Vue Devtools to work in production build is to use the `--debug` command line parameter.
```
quasar build --debug
```

**Comment by hawkeye64** [comment]
Fix will be in @quasar/app v2.2.0

**Comment by rstoenescu** [comment]
Those who've upgraded to q/app v2.1.2, please do a quick upgrade to v2.1.3.
process.env.DEBUG should be process.env.DEBUGGING.
